docs/usage.md

```diff
@@ -99,9 +100,11 @@ export default {
 };
 ```
 
-## With Svelte VS Code
+## With Svelte Config
 
-[svelte-vscode](https://marketplace.visualstudio.com/items?itemName=svelte.svelte-vscode) needs to know how its (svelte) language server should preprocess your files. This can be achieved by creating a `svelte.config.js` file at the root of your project which exports a svelte options object (similar to `svelte-loader` and `rollup-plugin-svelte`).
+Useful for [svelte-vscode](https://marketplace.visualstudio.com/items?itemName=svelte.svelte-vscode) which needs to know how its (svelte) language server should preprocess your files and [svelte-kit](https://github.com/sveltejs/kit) which can preprocess your svelte files prior to building.
```

Comment:
For me a little introductionary text (like it was before) is missing now. Like, "what is a Svelte config?"

Reply:
the introductory text is below... isn't it?